### PR TITLE
ci(dependabot): change to monthly interval

### PR DESCRIPTION
**Pull request by ErikSchierboom**
This PR changes the interval for dependabot updates to monthly.

Right now, the interval is usually set to daily, which results in (almost) daily notifications for Exercism maintainers, and frequent Docker updates to all tooling repos.

To reduce the maintenance burden, we are lowering the frequency to monthly. Due to Exercism's setup, it is unlikely that dependabot issues in our repositories need to be handled with any urgency, so we consider this to be a wise tradeoff.

For maintained track repos, maintainers can choose to close this PR unmerged. But for tooling repos and unmaintained tracks, we are requiring it to be merged.

Thanks!

### CODE REVIEW

**Review by IsaacG — APPROVED** [review]

**Comment by colinleach** [comment]
I'm unsure what to do with this and similar PRs. Merging is blocked, pending approval by `maintainers-admin`, but apparently that is still just Erik. Should this now be @exercism/cross-track-maintainers ?

**Comment by colinleach** [comment]
It lets me add myself as a reviewer, but I can't remove Erik.

**Comment by BNAndras** [comment]
That seems odd. I’ve merged quite a few of these PRs as a cross-track-maintainer member. What if we make a new PR with the changes from this PR and then close this one?

**Comment by colinleach** [comment]
Sure, whatever works.

Unfortunately this may also be necessary for PRs #767, #772, #773, #775. I'll make new PRs.

**Comment by IsaacG** [comment]
I asked about that here (restricted thread): https://forum.exercism.org/t/maintained-vs-single-maintained-vs-unmaintained-tracks/12636/69

**Comment by colinleach** [comment]
Did you get a reply? (I don't have read access to this thread)

**Comment by IsaacG** [comment]
> Did you get a reply? (I don't have read access to this thread)

Not yet. Jeremy and Erik are usually less active on the weekend.

**Comment by ErikSchierboom** [comment]
See https://github.com/exercism/julia/pull/778

**Pull request merged**
